src/dashboards/Compact/ScooterTile/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+
+import service from '../../../service'
+import { ScooterOperator, Scooter } from '@entur/sdk'
+import ScooterRow from './ScooterRow'
+
+import { ScooterIcon } from '@entur/icons'
+import { useSettingsContext } from '../../../settings'
+
+import ScooterOperatorLogo from '../../../assets/icons/scooterOperatorLogo'
+
+import './styles.scss'
+
+function ScooterTile({ scooters }: Props): JSX.Element {
+    const [settings] = useSettingsContext()
+
+    if (Object.entries(scooters)) {
```

Comment:
Unødvendig if-sjekk, for `Object.entries({})` tomt objekt) vil gi `[]` (tomt array) som evaluerer til true. Sjekk heller på `Object.entries(scooters).length > 0` i tilfelle.

Reply:
Ta gjerne også å snu om på if-setninga og returner tidlig, så ikkje alt er inni ein gigantisk if-statement. Då kan man droppe `else` også, sidan man returnerer uansett.

```
if (!Object.entries(scooters)) {
  return (
            <div>
                <h1>Laster inn scooters</h1>
            </div>
        )
}

return (
   <div className="scooterview">
...
```

```

Reply:
Den if sjekken er for å sjekke om det er undefined, ikke tomt array. Men kanskje du har en bedre måte å løse det på?

Reply:
Då må du sjekke før du brukar `Object.entries`, i tilfelle. Du kan default til tomt objekt. Og så kan du sjekke på length:

```
if (Object.entries(scooters || {}).length > 0) {
```

Spesifiser også i Props at `scooters` kan vere `undefined` eller `null` eller kva det kan vere.

Reply:
Done